src/components/ActivitiesList.js

```diff
@@ -23,7 +22,7 @@ ActivitiesList.propTypes = {
 };
 
 const mapStateToProps = state => ({
-  activities: selectActivities(state.activities, state.filters),
+  activities: state.activities.items,
```

Comment:
it is a good approach to have selectors. 
So, in selectors file for activities, you could create function `getActivitiesItems`, 
and use it here like: `getActivitiesItems(state)`. So, this function has to get state in argument.